coalib/bearlib/abstractions/Linter.py

```diff
@@ -196,6 +197,11 @@ def check_prerequisites(cls):
                         check_call(options["prerequisite_check_command"],
                                    stdout=DEVNULL,
                                    stderr=DEVNULL)
+                        warnings.simplefilter("default")
```

Comment:
I dislike the fact you need to turn on these warnings manually to see them (then we can also put them into the logging debug-channel btw, no need to utilize another module). The idea is actually that the developer can see what's wrong/not as it should be, and this should be fixed then as soon as possible. Especially for this kind of warning which is easy to fix for a developer I would still like to see a `logging.warn(...)`.
\+ users of coala are developers, though not explicitly for coala^^